plugins/modules/dcnm_fabric.py

```diff
@@ -37,6 +37,11 @@
         description:
         - The state of the feature or object after module completion
         type: str
+    skip_validation:
+        default: false
+        description:
+        - Skip playbook parameter validation.  Useful for debugging.
```

Comment:
If the user enables this flag but still passes invalid keys/values to NDFC, then we are relying on NDFC to return a meaningful error message in that case right?

Reply:
Yes, that's correct.